### ISSUE TITLE
change argument alters other defined rules 

### ISSUE THREAD

**Issue by dastergon** [thombashi/tcconfig#79]
According to the help page of tcset:
```
 --change               change existing traffic shaping rules to new one. this
                        option reduce the shaping rule switching side effect
                        (such as traffic spike) compared to --overwrite
                        option. note: the tcset command will fail when there
                        is no existing shaping rules.
```

## How to reproduce

Set multiple rules:
```
$ sudo tcset --device eth0 --delay 100 --rate 50k --network 192.168.1.2 --add
$ sudo tcset --device eth0 --delay 100 --rate 50k --network 192.168.1.3 --add
```
Show the existing rules:
```
$ sudo tcshow --device eth0
{
    "eth0": {
        "outgoing": {
            "dst-network=192.168.1.2/32, protocol=ip": {
                "delay": "100.0",
                "rate": "50K"
            },
            "dst-network=192.168.1.3/32, protocol=ip": {
                "delay": "100.0",
                "rate": "50K"
            }
        },
        "incoming": {}
    }
}
```

If I want to change the delay and rate in the dst-network 192.168.1.3  to 200 and 60k respectively:
```
$ sudo tcset --device eth0 --delay 200 --rate 60k --network 192.168.1.3 --change
```
However the command changes the rule for the dst-network 192.168.1.2:
```
$ sudo tcshow --device eth0
{                                              
    "eth0": {                               
        "outgoing": {                          
            "dst-network=192.168.1.2/32, protocol=ip": {                                       
                "delay": "100.0",              
                "rate": "60K"                  
            },                                 
            "dst-network=192.168.1.3/32, protocol=ip": {                                       
                "delay": "100.0",              
                "rate": "50K"                  
            }                                  
        },                                     
        "incoming": {}                         
    }                                          
}
```

The command replaced only the `rate` attribute in the wrong network and the `delay ` attribute was not changed at all.

Thanks in advance.

**Comment by thombashi** [comment]
Thank you for your report.

I can reproduce the problem.
That would be a great help for me.
I will fix the problem in this weekend.

**Comment by dastergon** [comment]
@thombashi is there an ETA regarding the fix of this issue?

**Comment by thombashi** [comment]
@dastergon
Sorry to have kept you waiting.
I've fixed the problem at `tcconfig 13.2`.

```
+ tcset --device ens33 --delay 100 --rate 50k --network 192.168.1.2 --overwrite
+ tcset --device ens33 --delay 100 --rate 50k --network 192.168.1.3 --add
+ tcshow -d ens33
{
    "ens33": {
        "outgoing": {
            "dst-network=192.168.1.2/32, protocol=ip": {
                "delay": 100,
                "rate": "50K"
            },
            "dst-network=192.168.1.3/32, protocol=ip": {
                "delay": 100,
                "rate": "50K"
            }
        },
        "incoming": {}
    }
}
+ tcset --device ens33 --delay 200 --rate 60k --network 192.168.1.3 --change
+ tcshow -d ens33
{
    "ens33": {
        "outgoing": {
            "dst-network=192.168.1.2/32, protocol=ip": {
                "delay": 100,
                "rate": "50K"
            },
            "dst-network=192.168.1.3/32, protocol=ip": {
                "delay": 200,
                "rate": "60K"
            }
        },
        "incoming": {}
    }
}
```

**Comment by dastergon** [comment]
Thanks for the update!
